### ISSUE TITLE
Please change to accessibilityRole from accessibilityTraits

### ISSUE THREAD

**Issue by RuairiOliver** [kirankalyan5/react-native-segmented-control-tab#155]
accessibilityTraits and accessibilityComponentType have been merged to use one option accessibilityRole since react native 0.57. TabOption should have accessibilityRole="button"

https://github.com/kirankalyan5/react-native-segmented-control-tab/issues/new

**Comment by kirankalyan5** [comment]
Hi @RuairiOliver, I'm quite occupied at the moment, please feel free to open a PR.
Thanks.
